language-design.md

```diff
@@ -62,7 +186,21 @@ the commenters?
 
 **Response**
 
+The conflict set up by the article and the comments is that professional programmers require robust tools with high specificity while everyone else may prefer languages that are higher-level abstractions. It seems that the premise of the article relies on the idea that most people should be able to understand code and that programming languages are not meeting their goals of being readable since people who do not program professionally cannot comprehend code. We disagree with this notion and side with the substance of the majority of the commenters' arguments. Programming should be something that is learned is not entirely intuitive to every English speaker. Just as English would not be intuitive to a Latin scholar, even though English is partially derived from Latin, programming languages are not intuitive to English speakers, even though many programs and languages are based in English. We believe that it is naive to think that programming languages can become intuitive to any English speaker, and that programming languages should be learned just like French and Spanish. Just like how there is overlapping grammar and vocabulary between Latin languages that ease fluency between speakers of those languages, we also believe this is how programmers quickly develop an ease of fluency between C derived languages. 
 
+Some of the commenters argue that making programming tools more accessible only makes it easier for people to learn how to progrtam, but not necessarily how to program _well_, which is important. While we understand their concerns and agree to them to some extent, there should be some achievable balance through which someone who is familiar with a language can express that idea in the form of a program, and someone who is not familiar with that language, or programming altogether, can understand the intent of the program. Perhaps the most common programming languages do not quite achieve that lofty goal, but as one commenter points out "languages like C++, Perl, Python, and Java have stood the test of time because they provide the right balance between granular control over the computer and powerful abstraction" [Eric]. 
+
+We also generally disagree with the article and the evidence brought up in the article since Perl is a difficult language to read in the first place. If they had compared a language that programmers typically start their programming experience such as Python, Processing, or Scratch, the results would be wildly different.
```

Comment:
I'd say every language serves a purpose. Some may claim that Perl is difficult to read, but it's great at scripting. It's a bit harsh to say that people who use it "hate themselves."